app/docker/views/containers/create/createContainerController.js

```diff
@@ -222,9 +236,22 @@ function ($q, $scope, $state, $timeout, $transition$, $filter, Container, Contai
       IPAMConfig: {
         IPv4Address: $scope.formValues.IPv4,
         IPv6Address: $scope.formValues.IPv6
-      }
+      },
     };
 
+    if (networkCanAlias(networkMode)) {
+      var aliases = [];
+      $scope.formValues.Aliases.forEach(function (v) {
```

Comment:
Use lodash forEarch (`_.forEarch()`) instead of ES5 buildin.

Reason: performance wise, the ES5 forEach is the worst of lodash, angular or ES5 forEach().